docs/release-notes/multirm-for-k8s.rst

```diff
@@ -8,7 +8,8 @@
    ``additional_resource_managers`` and ``resource_pools`` (additional resource managers are
    required to have at least one resource pool defined). Additional resource managers and their
    resource pools must have unique names. For more information, visit :ref:master configuration
-   <master-config-reference>.
+   <master-config-reference>. Support for notebooks and other workloads that require proxying is
+   currently under development.
```

Comment:
if we can, we should avoid saying "currently under development" in the release notes

Reply:
just "under development"? or what do you suggest?